autoload/EasyMotion/cmigemo.vim

```diff
@@ -92,7 +92,13 @@ function! EasyMotion#cmigemo#getMigemoPattern(input) "{{{
 
     if has('migemo')
         " Use migemo().
-        return migemo(a:input)
```

Comment:
Thank you for your review. I fixed it to use currenct &migemodict value if it exists.